src/pages/Home.vue

```diff
@@ -175,32 +268,98 @@ export default {
       return validCount == this.getLengthOfAuthType;
     },
 
-    /** TEMP - Apart from ID, are any extra inputs being validated. */
+    /** Apart from ID, are any extra inputs being validated.
+     * (THIS METHOD WILL BE DEPRECATED IN V2)
+     */
     isExtraInputValidationsRequired() {
       return this.getLengthOfAuthType > 1 && this.areAuthTypesValid;
     },
 
-    /** TEMP - Returns the auth methods used by each group */
+    /** Returns the auth methods used by each group.
+     * (THIS METHOD WILL BE DEPRECATED IN V2)
+     */
     getAuthType() {
       return this.groupData && this.groupData.authType
         ? this.groupData.authType
         : "ID";
     },
 
     /**
-     * Checks if the session type is a sign-in.
-     * @returns {boolean} True if the session type is a sign-in, false otherwise.
+     * Retrieves the authentication types.
+     * @returns {string[]} An array of authentication types.
+     */
+    getAuthTypes() {
+      return (
+        (this.sessionData && this.sessionData.auth_type.split(",")) ||
+        (this.auth_type && this.auth_type.split(",")) ||
+        (this.groupData && this.groupData.input_schema.auth_type.split(","))
+      );
+    },
+
+    /** Returns if sign up flow should be enabled */
+    isSignupEnabled() {
+      return (
+        (this.sessionData && this.sessionData.enable_signup == "True") ||
+        this.enable_signup == "True"
```

Comment:
when will this.enable_signup = True happen? its not being set anywhere?

Reply:
Its a URL parameter.